update_dockerfiles_on_merge.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
```

Comment:
Maybe you need to fix your editor ... this entire file uses spaces.

Reply:
updated the file to have tabs instead of spaces